plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1967,27 +1969,35 @@ elementgui.plant.spawn_world_types=Spawn in these dimensions (leave empty to dis
 elementgui.plant.error_plant_needs_name=Plant needs a name
 elementgui.plant.page_visual_and_type=Visual and type
 elementgui.plant.properties_advanced_plant=Advanced plant properties
-elementgui.potion.is_instant=Is instant
```

Comment:
I don't think it's needed to change the localization keys. Translators will also need to translate again all these texts.